editor/plugins/path_3d_editor_plugin.h

```diff
@@ -39,6 +39,18 @@
 
 class MenuButton;
 
+// Map handle id to control point id and handle type.
+enum HandleType {
+	HANDLE_TYPE_IN,
+	HANDLE_TYPE_OUT,
+	HANDLE_TYPE_TILT,
+};
+
+struct HandleInfo {
+	int point_idx; // Index of control point.
+	HandleType type; // Type of this handle.
+};
+
```

Comment:
They should be members of the Path3DGizmo class, not put it on global.

Reply:
done